File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        StringBuilder answer=  new StringBuilder();
        boolean isFirst = true;
        
        for(char c : s.toCharArray()){
            if( c== ' '){
                answer.append(c);
                isFirst = true;
            }
            else{
                if(isFirst){
                    answer.append(Character.toUpperCase(c));
                }
                    else{
                        answer.append(Character.toLowerCase(c));
                    }
                    isFirst = false;
                
            }
        }
        return answer.toString();
    }
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    var answer = '';
    var array = s.split('')
    array[0] = array[0].toUpperCase()
    for(var i = 1; i<array.length; i++){
        if(array[i-1] === ' '){
            array[i] = array[i].toUpperCase()
        }
        else{
            array[i] = array[i].toLowerCase()
        }
    }
    answer = array.join('')
    return answer;
}